inc/posttype/namespace.php

```diff
@@ -233,6 +233,16 @@ function after_title( $post ) {
 		_e( 'HTML and shortcodes are not supported in glossary terms.', 'pressbooks' );
 		echo '</p>';
 	}
+	if ( $post->post_type === 'back-matter' ) {
+		$taxonomy = \Pressbooks\Taxonomy::init();
+		if ( $taxonomy->getBackMatterType( $post->ID ) === 'glossary' ) {
+			echo '<div id="pb-post-type-notice" class="notice notice-info"><p>';
```

Comment:
```suggestion
			echo '<div id="pb-post-type-notice" class="notice notice-info" role="alert"><p>';
```

This should be an [ARIA live region](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) so that it will be announced to screen reader users when they switch the back matter type to Glossary.